conf/airframes/tudelft/guido_ardrone2_optitrack.xml

```diff
@@ -30,7 +30,7 @@ ARDrone2 with optical_flow landing.
     <module name="ins" type="extended"/>
 
     <module name="bat_voltage_ardrone2"/>
-    <module name="video_thread"/>
```

Comment:
is it really a bebop cam for an ardrone2 ?

Reply:
Whoops, thanks. I will still wait for @dewagter to take a look at it before I merge as it is quite a lot.